interface/forms/LBF/php/find_encounter_form_popup.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+/**
+ *
+ * Script to find encounter form for copy form field data
+ *
+ * @package   OpenEMR
+ * @link      https://www.open-emr.org
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+*/
+
+require_once("../../../globals.php");
+require_once("$srcdir/api.inc.php");
+require_once("$srcdir/forms.inc.php");
+require_once("$srcdir/options.inc.php");
+
+use OpenEMR\Core\Header;
+
+$dateFormat = DateFormatRead("jquery-datetimepicker");
+
+$pid = isset($_REQUEST['pid']) ? $_REQUEST['pid'] : "";
+$formname = isset($_REQUEST['formname']) ? $_REQUEST['formname'] : "";
+$encounter = isset($_REQUEST['encounter']) ? $_REQUEST['encounter'] : "";
+
+function fetch_appt_signatures_data_byId($eid)
+{
+    if (!empty($eid)) {
+        $eSql = "SELECT FE.encounter, E.id, E.tid, E.table, E.uid, U.fname, U.lname, E.datetime, E.is_lock, E.amendment, E.hash, E.signature_hash
+                FROM form_encounter FE
+                LEFT JOIN esign_signatures E ON (case when E.`table` ='form_encounter' then FE.encounter = E.tid else  FE.id = E.tid END)
+                LEFT JOIN users U ON E.uid = U.id
+                WHERE FE.encounter = ?
+                ORDER BY E.datetime ASC";
+        $result = sqlQuery($eSql, array($eid));
+        return $result;
+    }
+    return false;
+}
+
+$result4 = sqlStatement("SELECT fe.encounter,fe.date,openemr_postcalendar_categories.pc_catname, us.fname, us.mname, us.lname FROM form_encounter AS fe left join openemr_postcalendar_categories on fe.pc_catid=openemr_postcalendar_categories.pc_catid  left join users AS us on fe.provider_id = us.id  WHERE fe.pid = ? AND fe.encounter != ? order by fe.date desc", array($pid, $encounter));
+
+$enounterList = array();
+while ($rowresult4 = sqlFetchArray($result4)) {
+    $encounter = isset($rowresult4['encounter']) ? $rowresult4['encounter'] : '';
+    $id = '';
+
+    if (!empty($encounter)) {
+        $sql = "SELECT * FROM forms WHERE deleted=0 AND pid=? AND encounter=? AND formdir=?";
+        $parms = array($pid, $encounter, $formname);
+        $frow = sqlQuery($sql, $parms);
+        if ($frow['form_id']) {
+            $id = $frow['form_id'];
+        }
+    }
+
+    if (!empty($id)) {
+        $rowresult4['form_id'] = $id;
+        $enounterList[] = $rowresult4;
+    }
+}
+?>
+
+<html>
+<head>
+    <title><?php echo xl('Select Encounter'); ?></title>
+    <link rel="stylesheet" href='<?php echo $css_header ?>' type='text/css'>
```

Comment:
shouldn't need line above (below line does that and note that the below line does not need `dialog`, `jquery`, `fontawesome`, or `main-theme` in the parameters since these are always included)